lib/elixir/test/elixir/calendar/duration_test.exs

```diff
@@ -8,21 +8,21 @@ defmodule DurationTest do
     assert Duration.new!(year: 2, month: 1, week: 3) == %Duration{year: 2, month: 1, week: 3}
     assert Duration.new!(microsecond: {20000, 2}) == %Duration{microsecond: {20000, 2}}
 
-    assert_raise KeyError, ~s/key :months not found/, fn ->
```

Comment:
We're asserting on raised `KeyError`s in some of the calendar type tests we'll need to fix up as well